### PR TITLE
Zoinks!

### PR DESCRIPTION
**Pull request by isikkema**
Zoinks!

### CODE REVIEW

**Review by robobenklein — CHANGES_REQUESTED** [review]
Out of scope

File: hello.py [hunk 1]
```diff
@@ -6,6 +6,9 @@ def helloworld():
 def add_numbers(x, y):
     pass
 
+def zoinks():
+    print("Zoinks!")
```
Comment (robobenklein) [inline hello.py]: too exclamatory

File: hello.py [hunk 2]
```diff
@@ -6,6 +6,9 @@ def helloworld():
 def add_numbers(x, y):
     pass
 
+def zoinks():
```
Comment (robobenklein) [inline hello.py]: needs an argument here

**Review by robobenklein — CHANGES_REQUESTED** [review]

**Comment by isikkema** [comment]
Overruled.

**Pull request merged**